docs/code/MlNetCookBook.md

```diff
@@ -182,10 +183,10 @@ When the input file contains many columns of the same type, always intended to b
 
 [Example file](../../test/data/generated_regression_dataset.csv):
 ```
--2.75,0.77,-0.61,0.14,1.39,0.38,-0.53,-0.50,-2.13,-0.39,0.46,140.66
--0.61,-0.37,-0.12,0.55,-1.00,0.84,-0.02,1.30,-0.24,-0.50,-2.12,148.12
--0.85,-0.91,1.81,0.02,-0.78,-1.41,-1.09,-0.65,0.90,-0.37,-0.22,402.20
-0.28,1.05,-0.24,0.30,-0.99,0.19,0.32,-0.95,-1.19,-0.63,0.75,443.51
+-2.75;0.77;-0.61;0.14;1.39;0.38;-0.53;-0.50;-2.13;-0.39;0.46;140.66
```

Comment:
>; [](start = 5, length = 1)

Why are you changing the commas to semicolons?

Reply:
Ok, now I see that semicolon is actually the separator in this dataset. Should we change it to .txt instead of .csv?

---
In reply to: [370990496](https://github.com/dotnet/machinelearning/pull/4706#discussion_r370990496) [](ancestors = 370990496)

Reply:
I wouldn't touch this right now, with all the issues we are seeing with tests failing randomly.

---
In reply to: [370990675](https://github.com/dotnet/machinelearning/pull/4706#discussion_r370990675) [](ancestors = 370990675,370990496)